frontend/integration-tests/tests/monitoring.scenario.ts

```diff
@@ -1,10 +1,11 @@
-import { ExpectedConditions as until } from 'protractor';
+import {$, ExpectedConditions as until} from 'protractor';
```

Comment:
Rather than import `$`, let's add all the page elements we need to access to `monitoring.view.ts`.